src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -715,6 +715,39 @@ xclSKGetCmd(xclSKCmd *cmd)
   return ret;
 }
 
+int
+shim::
+xclAIEGetCmd(xclAIECmd *cmd)
+{
+  int ret;
+  drm_zocl_aie_cmd scmd;
+
+  ret = ioctl(mKernelFD, DRM_IOCTL_ZOCL_AIE_GETCMD, &scmd);
+
+  if (!ret) {
+    cmd->opcode = scmd.opcode;
+    cmd->size = scmd.size;
+    snprintf(cmd->info, scmd.size, "%s", scmd.info);
+  }
+
+  return ret;
+}
+
+int
+shim::
+xclAIEPutCmd(xclAIECmd *cmd)
+{
+  int ret;
+  drm_zocl_aie_cmd scmd;
+
+  scmd.opcode = cmd->opcode;
+  scmd.size = cmd->size;
+  snprintf(scmd.info, cmd->size, "%s",cmd->info);
+  ret = ioctl(mKernelFD, DRM_IOCTL_ZOCL_AIE_PUTCMD, &scmd);
+
+  return ret;
```

Comment:
How return return ioctl(...) and eliminate int ret?

Reply:
addressed